### DISCUSSION TITLE
milvus-operator seems support ingress and how to use it ?

### DISCUSSION THREAD

**Discussion by xiedeyantu** [milvus-io/milvus#26556] [Q&A and General discussion]
How to use ingress for milvus

**Comment by yhmo** [comment]
There are some discussions about milvus+ingress: https://github.com/milvus-io/milvus/discussions?discussions_q=is%3Aopen+ingress

**Reply by xiedeyantu** [reply]
Could you show me demo for milvus-operator ? I could not find answers following these discussions.
